README.md

```diff
@@ -63,6 +63,17 @@ Example:
 sg-otio write -u URL --session-token TOKEN --entity-type Cut --entity-id CUT_ID --file INPUT.edl --movie INPUT.mov --settings SETTINGS.JSON
 ```
 
+### Comparing a Video Track to a SG Cut
+Read a Video Track from an OpenTimelinio source and compare it to an existing SG Cut.
+Any format suppported by OpenTimelineIO's standard adapters is supported for the source.
+The video Track can be written to SG as a new Cut by adding the `--write` argument.
```

Comment:
```suggestion
The video Track can be written to SG as a new Cut by adding the `--write` argument.
If the `--write` argument is not provided, the result of the comparison is returned.
```

Reply:
Hmm not really. The result of the comparison is written to the output, regardless of the `--write` argument.